src/api/controllers/user/user.login.js

```diff
@@ -39,13 +39,13 @@ module.exports = app => {
     const { User, Network } = req.app.locals.models
 
     try {
-      const username = req.body.name
+      const username = req.body.username
       const password = req.body.password
 
       // Get user
       const user = await User.findOne({
         where: {
-          [Op.or]: [{ name: username }, { email: username }]
+          [Op.or]: [{ username: username }, { email: username }]
```

Comment:
```suggestion
          [Op.or]: [{ username }, { email: username }]
```

Car clé = valeur donc pas nécessaire de répéter

Reply:
Grosse erreur que j'ai faite ! Pourtant, ça passait sur Postman mdr. J'avais du me grourer quelque part ^^

Reply:
Ca marche pas avec juste { username } ?

Reply:
Si si, bien sûr, tkt ^^